quote-book/src/in_memory_quote_book.rs

```diff
@@ -12,7 +12,7 @@ use std::{
 };
 
 /// A naive in-memory quote book implementation
-#[derive(Clone, Debug)]
+#[derive(Clone)]
```

Comment:
Curious why was `Debug` removed?

Reply:
I think this was because previously I had added a ledger in here which got rid of debug, but now that I moved that out, this change shouldn't be here. Thank you.